### PR TITLE
Homepage - Header section 

### PR DESCRIPTION
**Pull request by lu-jim**
## First PR for this project

- Configured both the structure and styling for the header.
- Added linters
- Customized README

### CODE REVIEW

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @lu-jim, You did a great job so far 👏🏼 👏🏼 

- Well done on following the original design
- Well done on using HTML semantic tags
- The linter checks are all passing
- The Readme file is descriptive

However, here are some points that you need to work on to get your project approved.

## Required changes :

- [ ] Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <a href="#" class="logo">My Logo</a>
+        <nav class="hamburger-menu"><a href="#">☰</a></nav>
+        <ul class="menu">
+          <li class="menu-item">Portfolio</li>
+          <li class="menu-item">About</li>
+          <li class="menu-item">Contact</li>
+        </ul>
+      </nav>
+
+      <section class="content">
```
[the remark this answers is not in the document]
Comment (lu-jim) [reply index.html]: Hello! Thanks for the feedback. Anything else I should fix before the second review?

[the remark this answers is not in the document]
Comment (lu-jim) [reply index.html]: I just noticed I have first a nav section and then a header section with the headline in my fix. Should the header only be for the navbar and another section just for the headline?

**Review by carlos-ssh — APPROVED** [review]
Hi @lu-jim , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**